File: IGS/2022/20212022-S2/11KOM1/03_JAVA/00_intro/Main.java
public class Main {

	public static void main(String[] args) {
		System.out.println("Hello, world!");
		if (args.length > 0){
			// for (int index = 0; index < args.length; index++){

			// 	System.out.println( index + " " + args[index] );

			// }

			for (String arg : args) {
				System.out.println(arg);
			}
		}
	}

}

File: IGS/2022/20212022-S2/11KOM1/03_JAVA/00_intro/main.py
from sys import argv

class Main:

	def main(self):
		print("Hello, world!")
		if len(argv) > 1:
			for i in range(1,len(argv)):
				print(i, argv[i])


if __name__ == "__main__":
	app = Main()
	app.main()